akka-stream/src/main/scala/akka/stream/impl/fusing/IteratorInterpreter.scala

```diff
@@ -145,7 +145,8 @@ private[akka] class IteratorInterpreter[I, O](val input: Iterator[I], val ops: S
       outHandlers,
       logics,
       (_, _, _) ⇒ throw new UnsupportedOperationException("IteratorInterpreter does not support asynchronous events."),
-      fuzzingMode = false)
+      fuzzingMode = false,
+      null)
```

Comment:
well, it might, just don't mistakenly compare them to each other and think they represent the same execution island.